build/ftest-base/pom.xml

```diff
@@ -39,19 +39,10 @@
     <arquillian.launch.glassfish40>false</arquillian.launch.glassfish40>
   </properties>
 
-  <dependencies>
-    <dependency>
-      <groupId>org.jboss.shrinkwrap.resolver</groupId>
-      <artifactId>shrinkwrap-resolver-depchain</artifactId>
-      <version>${version.shrinkwrap.resolver}</version>
-      <scope>test</scope>
-      <type>pom</type>
-    </dependency>
-  </dependencies>
-
```

Comment:
the above dependency is already available indirectly via arquillian-drone-webdriver-depchain, which is fine. So deleting it should be fine.